dashboard/test/models/user_test.rb

```diff
@@ -1597,20 +1597,20 @@ def assert_fails_email_uniqueness_validation(user)
     assert_equal [s2, s1, c], user.user_scripts
     assert_equal [s2.script, s1.script, c.script], user.scripts
 
-    # working on scripts
-    assert_equal [s2.script, s1.script], user.working_on_scripts
     # primary script -- most recently progressed in
+    assert_equal s2.script, user.script_with_most_recent_progress
     assert_equal s2.script, user.primary_script
 
     # add an assigned script that's more recent
-    a = create :user_script, user: user, started_at: (Time.now - 1.day)
-    assert_equal [a.script, s2.script, s1.script], user.working_on_scripts
+    a = create :user_script, user: user, assigned_at: (Time.now - 1.day)
+    assert_equal a.script, user.most_recently_assigned_script
     assert_equal a.script, user.primary_script
 
-    # make progress on an older script
+    # make progress on a older script, assignment is still primary
```

Comment:
"an older" is grammatically correct, so please don't change it to "a older".

I like this improvement to the comments, but I'm not totally sure I follow -- what does "assignment is still primary" mean?